### PR TITLE
44-Add a Timetable entity

### PR DESCRIPTION
**Pull request by mohsenmansourvar**

### CODE REVIEW

File: src/main/java/com/education/domain/TimeTable.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import java.util.List;
+
+@Entity
+public class TimeTable {
+    @Id
+    @GeneratedValue
+    private Long id;
+    @Temporal(TemporalType.TIMESTAMP)
+    private LocalTime from;
+    @Temporal(TemporalType.TIME)
```
Comment (mohsenebrahimi123) [inline src/main/java/com/education/domain/TimeTable.java]: Use TIMESTAMP instead of TIME

File: src/main/java/com/education/domain/TimeTable.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    public List<Student> getStudents() {
+        return students;
+    }
+
+    public void setStudents(List<Student> students) {
+        this.students = students;
+    }
+}
```
Comment (mohsenebrahimi123) [inline src/main/java/com/education/domain/TimeTable.java:102]: Override equals method

Comment (mohsenebrahimi123) [inline src/main/java/com/education/domain/TimeTable.java:102]: Override toString method

Comment (mohsenmansourvar) [reply src/main/java/com/education/domain/TimeTable.java:102]: Done

Comment (mohsenmansourvar) [reply src/main/java/com/education/domain/TimeTable.java:102]: Done

**Pull request merged**